go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.1
-	github.com/cert-manager/cert-manager v1.11.2
+	github.com/cert-manager/cert-manager v1.11.0
```

Comment:
downgrade due to this not fixed https://github.com/cert-manager/cert-manager/issues/5953